drivers/windows/labels.go

```diff
@@ -30,4 +30,10 @@ const (
 
 	// SourceMac of the network
 	SourceMac = "com.docker.network.windowsshim.sourcemac"
+
+	// EnableICC label
+	EnableICC = "com.docker.network.bridge.enable_icc"
```

Comment:
Ok I will rename these labels to some windows specific driver labels. Basically I want to have endpoint options to disable ICC and DNS on an endpoint.